.asf.yaml

```diff
@@ -21,16 +21,23 @@ github:
   collaborators:
     - krlmlr
     - nbenn
-    - ywc88
```

Comment:
Solomon hasn't contributed in a long time (and seems to have deleted/renamed the account since anyways)